deployment/ansible/inventory/static

```diff
@@ -0,0 +1,5 @@
+[tag_seed_uat]
+52.10.46.160
```

Comment:
I understand this file labels these 4 hosts with a group that then defines var `peer_keypairs`. Who guarantees that Ansible actually picks up this label for these 4 hosts? E.g. is it possible inventory is iterated over by host name rather than IP hence making this label not to be picked up (hence keys not instated if hosts created from scratch)?

Reply:
Good question, didn't found an explicit explanation in the docs, but I guess one can create inventory groups either by hostname or IP addresses.

This is tested already. Both in combination with dynamic inventory and this inventory alone.